orchagent/vnetorch.h

```diff
@@ -246,11 +247,12 @@ class VNetOrch : public Orch2
 const request_description_t vnet_route_description = {
     { REQ_T_STRING, REQ_T_IP_PREFIX },
     {
-        { "endpoint",    REQ_T_IP },
-        { "ifname",      REQ_T_STRING },
-        { "nexthop",     REQ_T_STRING },
-        { "vni",         REQ_T_UINT },
-        { "mac_address", REQ_T_MAC_ADDRESS },
+        { "endpoint",               REQ_T_IP_LIST },
+        { "ifname",                 REQ_T_STRING },
+        { "nexthop",                REQ_T_STRING },
+        { "vni",                    REQ_T_STRING },
```

Comment:
I do not think it would, since we will convert it to the right format. I think the only difference is to process the values in the request parser or vnetorch.